files/Orchestrator_performance.json

```diff
@@ -50,8 +51,6 @@
             "mode": "palette-classic"
           },
           "custom": {
-            "axisCenteredZero": false,
-            "axisColorMode": "text",
```

Comment:
what's the purpose of all those axis attributes and why are they gone?

Reply:
I'm not sure why small changes like this show up in the export.  All I did was make sure I had the latest dashboard from `main` on my box and add the new panel.  Maybe it's a Grafana version thing?